lib/styles/settings/_survey-toggle.scss

```diff
@@ -0,0 +1,54 @@
+@import '../settings/colors/tokens';
+
+$survey-toggle-size: 48px;
```

Comment:
why did you decided to extract it to a separate file?

Reply:
After discussion I removed all external settings for components (except Button).